src/Core/Domain/Product/Command/UpdateProductCommand.php

```diff
@@ -193,6 +195,46 @@ class UpdateProductCommand
      */
     private $reference;
 
+    /**
+     * @var DecimalNumber|null
+     */
+    private $width;
+
+    /**
+     * @var DecimalNumber|null
+     */
+    private $height;
+
+    /**
+     * @var DecimalNumber|null
+     */
+    private $depth;
+
+    /**
+     * @var DecimalNumber|null
+     */
+    private $weight;
+
+    /**
+     * @var DecimalNumber|null
+     */
+    private $additionalShippingCost;
+
+    /**
+     * @var DeliveryTimeNoteType
+     */
+    private $deliveryTimeNoteType;
+
+    /**
+     * @var string[]|null
+     */
+    private $localizedDeliveryTimeInStockNotes;
+
+    /**
+     * @var string[]|null
+     */
+    private $localizedDeliveryTimeOutOfStockNotes;
+
```

Comment:
no the carriers was moved to a dedicated command (so that we keep the "relations" in separate commands and only all the basicproduct fields in this unified command)

Reply:
ok